docs/src/gui/qtvcp-vismach.adoc

```diff
@@ -127,123 +138,159 @@ c.ready()
-* `box = BoxCenteredXY(xw, yw, z)` +
-  Creates a box of width xw / yw and height z.
+*`box = BoxCentered(xw, yw, zw)`*::
+  Creates an xw by yw by zw _box centred on the origin_.
+*`box = BoxCenteredXY(xw, yw, z)`*::
+  Creates a _box ground on WY plane_ of width xw / yw and height z.
 
 Composite parts may be created by assembling these primitives either at
 creation time or subsequently:
 
+[source,python]
 ----
 part1 = Collection([Sphere(100,100,100,50), CylinderX(100,40,150,30)])
 part2 = Box(50,40,75,100,75,100)
 part3 = Collection([part2, TriangleXY(10,10,20,10,15,20,100,101)])
 part4 = Collection([part1, part2])
 ----
 
-== Orientating Model Parts
+== Moving Model Parts
 
 Parts may need to be moved in the Vismach space to assemble the model.
+
+//FIXME unclear
 They may also need to be moved to create the animation as the animation
 rotation axis is created at the origin (but moves with the Part).
 
-* `part1 = Translate([part1], x, y, z)` +
+=== Translating Model parts
+
+*`part1 = Translate([part1], x, y, z)`*::
   Move part1 the specified distances in x, y and z.
-* `part1 = Rotate([part1], theta, x, y, z)` +
+
+=== Rotating Model Parts
+
+//FIXME angle unit ?
+*`part1 = Rotate([part1], theta, x, y, z)`*::
   Rotate the part by angle theta about an axis between the origin and
   x, y, z.
 
 == Animating Parts
 
-To animate the model (controlled by the values of HAL pins) there are
-two functions 'HalTranslate', 'HalRotate' and 'HalToolCylinder'. +
-For parts to move inside an assembly they need to have their HAL motions
-defined before being assembled with the "Collection" command. +
-The rotation axis and translation vector move with the part as it is
-moved by the vismach script during model assembly, or as it moves in
-response to the HAL pins as the model is animated.
-
-* `part = HalTranslate([part], None, "hal_pin_name", xs, ys, zs)` +
-  The function arguments are:
-** first a _collection/part_ which can be pre-created earlier in the
-  script, or could be created at this point if preferred eg
-  `part1 = HalTranslate([Box(....)], ...)`. +
-** The _HAL component_ is the next argument. +
-   In QtVCP if you are reading system pins directly then the component
-   argument is set to 'None". +
-** After that is the _name of the HAL pin_ that will animate the motion. +
-   This needs to match an existing HAL pin that describes the joint
-   position such as:
+//FIXME 2 or 3 functions ? HalToolCylinder not documented here ?
+To *animate the model controlled by the values of HAL pins* there are
+two functions `HalTranslate`, `HalRotate` and `HalToolCylinder`.
+
+_For parts to move inside an assembly they need to have their HAL motions
+defined before being assembled with the "Collection" command_.
+
+The *rotation axis and translation vector move with the part*:
+
+* as it is moved by the Vismach script during model assembly, or
+* as it moves in response to the HAL pins as the model is animated.
+
+=== `HalTranslate`
```

Comment:
In general I think we shouldn't have a mono-space font in numbered headings. But it's more a personal opinion.

Reply:
Like I said in another message, mono-space font looks adequate to me to style all commands, paths, files, functions, keywords... But the styles for titles should be improved to make it smoother to the eyes, like having the same styling as the rest of the title to start with